pages/common/ld.md

```diff
@@ -0,0 +1,16 @@
+# ld
+
+> The GNU ELF linker.
+> More information: <https://ftp.gnu.org/old-gnu/Manuals/ld-2.9.1/html_mono/ld.html>.
+
+- Link an object file with no dependencies into an executable:
+
+`ld {{path/to/file.o}} --output {{path/to/output_executable}}`
+
+- Link two object files together:
+
+`ld {{path/to/file1.o}} {{path/to/file2.o}} --output {{path/to/output_executable}}`
+
+- Dynamically link a 64-bit program to glibc (file paths change depending on system):
```

Comment:
```suggestion
- Dynamically link the 64-bit program to glibc (file paths change depending on system):
```

Reply:
Hi, there is a small error here. It is more grammatically correct to use `a` in the context of what is said here, since `64-bit program` is not being defined and is more generic therefore using `a` is more applicable @

Reply:
I don't know, how you pronounce `64-bit`, but I'd use `a` instead of `an`.

Reply:
This should be left as-is.

Ref https://www.merriam-webster.com/words-at-play/is-it-a-or-an

Reply:
:facepalm: rookie mistake, I don't think I proof-read

Reply:
Actually nevermind, there was a commit to make it x64..... therefore an does work and is correct.